pkgs/applications/editors/geany/default.nix

```diff
@@ -13,8 +15,9 @@ stdenv.mkDerivation rec {
   };
 
   NIX_LDFLAGS = if stdenv.isDarwin then "-lintl" else null;
-
-  buildInputs = [ gtk2 which pkgconfig intltool file ];
+  
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ gtk2 which intltool file ];
```

Comment:
@Mic92 was thinking it may be, will fix it :) running nox-review right now before I merge